goplay/src/main/java/be/tapped/goplay/content/ProgramRepo.kt

```diff
@@ -59,64 +48,33 @@ internal class HtmlFullProgramParser(private val jsoupParser: JsoupParser) {
             .flatMap {
                 Either.catch {
                     val programDataObject = Json.decodeFromString<JsonObject>(it)["data"]!!.jsonObject
-                    Json {
-                        isLenient = true
-                        ignoreUnknownKeys = true
-                    }.decodeFromJsonElement<Program>(programDataObject)
+                    jsonSerializer.decodeFromJsonElement<Program>(programDataObject)
                 }.mapLeft(Failure::JsonParsingException)
             }
 }
 
-// The Search API from Vier is sometimes returning non available Programs.
-// A redirect is triggered to a fixed location
-// Model this into a soft error
-internal class ProgramResponseValidator {
-    private companion object {
-        private const val NO_LONGER_AVAILABLE_REDIRECT_LOCATION = "$siteUrl/programma-niet-meer-beschikbaar"
-    }
-
-    internal suspend fun validateResponse(response: Response): Either<Failure, String> =
-        if (response.priorResponse?.headers("Location")?.firstOrNull() == NO_LONGER_AVAILABLE_REDIRECT_LOCATION) {
-            Failure.Content.ProgramNoLongerAvailable.left()
-        } else {
-            response.safeBodyString()
-        }
-}
-
 public sealed interface ProgramRepo {
-
     public suspend fun fetchPrograms(): Either<Failure, Success.Content.Programs>
-
     public suspend fun fetchProgram(programSearchKey: SearchHit.Source.SearchKey.Program): Either<Failure, Success.Content.SingleProgram>
-
 }
 
 internal class HttpProgramRepo(
-    private val client: OkHttpClient,
-    private val htmlProgramParser: HtmlProgramParser,
-    private val htmlFullProgramParser: HtmlFullProgramParser,
-    private val programResponseValidator: ProgramResponseValidator,
+    private val client: HttpClient = ktorClient,
+    private val htmlProgramParser: HtmlProgramParser = HtmlProgramParser(),
 ) : ProgramRepo {
 
-    // Scrapes the https://www.goplay.be/programmas searching for all available Programs and the details associated with it.
-    // Fortunately for us the details of a Program is encoded in the HTML DOM as a JSON
+    // Scrapes the https://www.goplay.be/programmas HTML content
     // curl -X GET "https://www.goplay.be/programmas"
-    override suspend fun fetchPrograms(): Either<Failure, Success.Content.Programs> = withContext(Dispatchers.IO) {
-        either {
-            val html = client.executeAsync(Request.Builder().get().url("$siteUrl/programmas").build()).safeBodyString().bind()
-            val programs = htmlProgramParser.parse(html).bind()
-            Success.Content.Programs(programs)
+    override suspend fun fetchPrograms(): Either<Failure, Success.Content.Programs> =
+        withContext(Dispatchers.IO) {
+            either {
+                val html = client.get<HttpResponse>("$siteUrl/programmas").readText()
```

Comment:
I'm using `bodyAsText` in another project which returns me an unescaped HTML string. It uses `CharSet.UTF_8` as a default param to `bodyAsTex`.

Reply:
Ooooooh 🙈
Let's take a look

Reply:
Wait I don't see any `bodyAsText ` function.

Either way the `readText` function returns it's content correctly.  
It's the Json that is embedded within the JSON that seems to be HTML encoded.

Reply:
```html
<div data-hero="{&quot;data&quot;:{&quot;id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;title&quot;:&quot;wtFOCK&quot;,&quot;subtitle&quot;:&quot;&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten.&quot;,&quot;brand&quot;:&quot;goplay&quot;,&quot;category&quot;:&quot;Fictie&quot;,&quot;label&quot;:&quot;Volledige reeks&quot;,&quot;link&quot;:&quot;\/wtfock&quot;,&quot;images&quot;:{&quot;hero&quot;:&quot;https:\/\/images.goplay.be\/meta\/wallpaper3840x2160wtfocks5-3-qvi8qv.jpg?auto=format&amp;crop=edges&amp;dpr=2&amp;fit=crop&amp;h=1080&amp;ixlib=php-1.1.0&amp;q=90&amp;w=1920&amp;s=13c3cece404ff434ca7d182416b47e4d&quot;,&quot;mobile&quot;:&quot;https:\/\/images.goplay.be\/2021-06\/screenshot-2021-06-25-at-151356-qvb54q.png?auto=format&amp;crop=edges&amp;dpr=2&amp;fit=crop&amp;h=500&amp;ixlib=php-1.1.0&amp;q=90&amp;w=375&amp;s=e77d788451f6a706bb2f4415e449b115&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;teaser&quot;:&quot;https:\/\/images.goplay.be\/meta\/wallpaper3840x2160wtfocks5-3-qvi8qv.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=max-185&amp;ixlib=php-1.1.0&amp;q=85&amp;w=335&amp;s=fcf45da4d87ae00fb6cc0b7291b84aff&quot;},&quot;header&quot;:{&quot;title&quot;:&quot;&quot;,&quot;video&quot;:[]},&quot;needs16PlusLabel&quot;:true,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten.&quot;,&quot;nodeId&quot;:&quot;14498&quot;,&quot;nodeUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1553702952,&quot;title&quot;:&quot;wtFOCK&quot;,&quot;type&quot;:&quot;program&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/wtfock&quot;},&quot;playlists&quot;:[{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1619215140,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1450,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:1,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 1&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-04-22-20h37m44s881-qrzafu.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=798547a05967ca423b5e3144fd0241b0&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-1&quot;,&quot;midrollOffsets&quot;:[844],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17011&quot;,&quot;nodeUuid&quot;:&quot;f060c406-98f7-4842-93b6-e55275454a3a&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1619215140,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 1&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901224740,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-1&quot;},&quot;pageUuid&quot;:&quot;f060c406-98f7-4842-93b6-e55275454a3a&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1619216590,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 1&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901224740,&quot;videoUuid&quot;:&quot;635f92fe-aa82-43be-8c87-70c790992af3&quot;,&quot;whatsonId&quot;:&quot;10142736556303527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1619819940,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1615,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:2,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 2&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-04-30-08h28m00s458-qsd5zc.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=0ad471d839a53aa95baff2f74c6abf0e&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-2&quot;,&quot;midrollOffsets&quot;:[897],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17215&quot;,&quot;nodeUuid&quot;:&quot;15bcdcc9-7812-4b28-9ecc-05bd2a3e8415&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1619819940,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 2&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-2&quot;},&quot;pageUuid&quot;:&quot;15bcdcc9-7812-4b28-9ecc-05bd2a3e8415&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1619821555,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 2&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;0069d6ac-e5ac-48db-be42-06583cb9830f&quot;,&quot;whatsonId&quot;:&quot;10142736556472527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;Nieuwe aflevering&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1620424740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1388,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:3,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 3&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-05-07-08h23m39s481-qsq4g3.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=e34f3c7610f316e70779c14a2b65c624&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-3&quot;,&quot;midrollOffsets&quot;:[668],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Axel.Florizoone@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17437&quot;,&quot;nodeUuid&quot;:&quot;b45e60de-723a-47b2-b301-58ee0c45ac1e&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1620424740,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 3&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-3&quot;},&quot;pageUuid&quot;:&quot;b45e60de-723a-47b2-b301-58ee0c45ac1e&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1620426128,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 3&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;7dcfc62e-ace2-4c30-b5c6-9f57d4a66ad8&quot;,&quot;whatsonId&quot;:&quot;10142736556481527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1621029540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1946,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:4,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 4&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-05-14-09h50m15s363-qt374i.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=259af36b5be9023a2c550be16198325f&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-4&quot;,&quot;midrollOffsets&quot;:[885],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17618&quot;,&quot;nodeUuid&quot;:&quot;043dfb74-1d15-4ef0-8d98-2a5555e9f680&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1621029540,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 4&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-4&quot;},&quot;pageUuid&quot;:&quot;043dfb74-1d15-4ef0-8d98-2a5555e9f680&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1621031486,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 4&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;8eed4181-c06a-4bac-8ab2-c0abb72678ad&quot;,&quot;whatsonId&quot;:&quot;10142736556490527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1621634340,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1689,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:5,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 5&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-05-21-08h10m22s061-qtg15z.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=dfcfc4573e27fa98f39f90bfce2e22b6&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-5&quot;,&quot;midrollOffsets&quot;:[895],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17793&quot;,&quot;nodeUuid&quot;:&quot;d6998574-b295-4b64-bd31-af5530855c0d&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1621634340,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 5&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-5&quot;},&quot;pageUuid&quot;:&quot;d6998574-b295-4b64-bd31-af5530855c0d&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1621636029,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 5&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;3db3c43f-ba10-46bb-9923-986ad3637e1b&quot;,&quot;whatsonId&quot;:&quot;10142736556499527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1622239140,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1194,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:6,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 6&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-05-27-14h07m55s802-qtrlq1.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=a85c06016baf92d70d975f9c997e9028&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-6&quot;,&quot;midrollOffsets&quot;:[603],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;17982&quot;,&quot;nodeUuid&quot;:&quot;6447369a-0eb3-4633-b677-250e41e2a4da&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1622239140,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 6&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-6&quot;},&quot;pageUuid&quot;:&quot;6447369a-0eb3-4633-b677-250e41e2a4da&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1622240334,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 6&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;c3a80f25-7ceb-4c11-ba46-c74299b45b9e&quot;,&quot;whatsonId&quot;:&quot;10142736556508527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1622843940,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1190,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:7,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 7&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-06-04-08h13m00s591-qu5ymg.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=02f27706d6e52b66436a0ae1d0ddc14c&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-7&quot;,&quot;midrollOffsets&quot;:[596],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;18159&quot;,&quot;nodeUuid&quot;:&quot;b2d38dcb-4e0e-47df-8cc9-3b12f4b71cc3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1622843940,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 7&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-7&quot;},&quot;pageUuid&quot;:&quot;b2d38dcb-4e0e-47df-8cc9-3b12f4b71cc3&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1622845130,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 7&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;1751d189-9a71-432f-ab69-38ed8c99f5f0&quot;,&quot;whatsonId&quot;:&quot;10142736556517527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1623448740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1927,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:8,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 8&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-06-11-08h20m52s383-quixo7.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=2ba37e7ba44e9937b7af3d3095ef6c20&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-8&quot;,&quot;midrollOffsets&quot;:[995],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;18314&quot;,&quot;nodeUuid&quot;:&quot;81ca6af5-750b-4a79-aa3b-9194274e1fa5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1623448740,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 8&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-8&quot;},&quot;pageUuid&quot;:&quot;81ca6af5-750b-4a79-aa3b-9194274e1fa5&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1623450667,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 8&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;273d27a5-4d0e-40d7-a4d5-05ba3926d806&quot;,&quot;whatsonId&quot;:&quot;10142736556526527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1624053540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1381,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:9,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 9&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-06-15-14h52m27s513-ququg5.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=443f5999b9339ba6204bb44d571bddd4&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-9&quot;,&quot;midrollOffsets&quot;:[788],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;18443&quot;,&quot;nodeUuid&quot;:&quot;1e59a89c-4299-4ffc-9eeb-7640b3606f7b&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1624053540,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 9&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-9&quot;},&quot;pageUuid&quot;:&quot;1e59a89c-4299-4ffc-9eeb-7640b3606f7b&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1624054921,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 9&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;2a78af3f-8a07-4c74-8f03-d7bcab74709e&quot;,&quot;whatsonId&quot;:&quot;10142736556535527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1624658340,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:2073,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:10,&quot;episodeTitle&quot;:&quot;S5 - Aflevering 10&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/screenshot-2021-06-25-at-101448-qvaw9y.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=62413993231e72ca8d240b1a545f7d6f&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-10&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Axel.Florizoone@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;18535&quot;,&quot;nodeUuid&quot;:&quot;05100750-a745-4793-b8bc-ba3069ee15de&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1624658340,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 10&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1901141940,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5\/wtfock-s5-aflevering-10&quot;},&quot;pageUuid&quot;:&quot;05100750-a745-4793-b8bc-ba3069ee15de&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:5,&quot;seekableFrom&quot;:1624660413,&quot;title&quot;:&quot;wtFOCK - S5 - Aflevering 10&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S5 - Aflevering 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1901141940,&quot;videoUuid&quot;:&quot;2cd98041-1251-4929-9b79-1ea4b5ba9d72&quot;,&quot;whatsonId&quot;:&quot;10142736556544527&quot;}],&quot;id&quot;:&quot;ad286182-f2fa-4c2b-8812-138b0503a229&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s5&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;michiel.vandevelde@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;16882&quot;,&quot;nodeUuid&quot;:&quot;ad286182-f2fa-4c2b-8812-138b0503a229&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1618862184,&quot;title&quot;:&quot;wtFOCK (voorjaar 2021)&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s5&quot;},&quot;title&quot;:&quot;Seizoen 5 - Yasmina&quot;},{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1599256740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1373,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:1,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 1&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-09-04-08h56m59s334-qg4gn5-qg4gn5.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=3672189b6e41778bf78ed494a588e5f9&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-1&quot;,&quot;midrollOffsets&quot;:[804],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14532&quot;,&quot;nodeUuid&quot;:&quot;b875a341-15e3-4e0b-acd3-61613aaba89b&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1599256740,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 1&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-1&quot;},&quot;pageUuid&quot;:&quot;b875a341-15e3-4e0b-acd3-61613aaba89b&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1599258113,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 1&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;86bc92fe-c686-4c53-995d-25ee830e6ded&quot;,&quot;whatsonId&quot;:&quot;10142736504656527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1599861540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1203,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:2,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 2&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-09-11-08h36m05s434-qgheeg-qgheeg.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=0a939afb1f9710a5e9636283a241795b&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-2&quot;,&quot;midrollOffsets&quot;:[631],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14533&quot;,&quot;nodeUuid&quot;:&quot;518d2fec-60b4-4f9f-9e37-9825421ead00&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1599861540,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 2&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-2&quot;},&quot;pageUuid&quot;:&quot;518d2fec-60b4-4f9f-9e37-9825421ead00&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1599862743,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 2&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;1c251dcf-eeb2-4055-ad8a-453415e9b812&quot;,&quot;whatsonId&quot;:&quot;10142736504795527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1600466340,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1176,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:3,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 3&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-09-18-08h25m57s852-qgucju-qgucju.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=4319af470c5f97d290e1d6c7554b3cf7&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-3&quot;,&quot;midrollOffsets&quot;:[584],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14534&quot;,&quot;nodeUuid&quot;:&quot;1b6c8f78-5232-4c93-bd41-857bb84c04bb&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1600466340,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 3&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-3&quot;},&quot;pageUuid&quot;:&quot;1b6c8f78-5232-4c93-bd41-857bb84c04bb&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1600467516,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 3&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;c573fba1-1951-4f97-a19a-bbae155291d1&quot;,&quot;whatsonId&quot;:&quot;10142736504804527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1601071140,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:983,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:4,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 4&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-09-25-07h48m59s146-qh79i2-qh79i2.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=a4f02cbaecfb4c6f8e7c002d8660fb7e&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-4&quot;,&quot;midrollOffsets&quot;:[521],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14535&quot;,&quot;nodeUuid&quot;:&quot;74441bf3-4a80-49f1-909d-e7dfc0227d51&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1601071140,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 4&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-4&quot;},&quot;pageUuid&quot;:&quot;74441bf3-4a80-49f1-909d-e7dfc0227d51&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1601072123,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 4&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;7312d64f-7345-4e76-ae8f-2f8dff7be169&quot;,&quot;whatsonId&quot;:&quot;10142736504813527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1601712000,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1339,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:5,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 5&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-10-02-09h02m24s511-qhkbnx-qhkbnx.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=afe3c46013a7da7591f677b20cf7779f&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-5&quot;,&quot;midrollOffsets&quot;:[724],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14536&quot;,&quot;nodeUuid&quot;:&quot;731e49b4-270e-4968-af0b-60ddc1ed094a&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1601712000,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 5&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-5&quot;},&quot;pageUuid&quot;:&quot;731e49b4-270e-4968-af0b-60ddc1ed094a&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1601713339,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 5&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;872cdb9f-5514-49c8-bbd6-e7c33490d354&quot;,&quot;whatsonId&quot;:&quot;10142736504822527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1602280740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1607,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:6,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 6&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-10-09-08h40m18s127-qhx98y-qhx98y.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=892099ebd10210f070ee4f07f1019a4e&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-6&quot;,&quot;midrollOffsets&quot;:[904],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14537&quot;,&quot;nodeUuid&quot;:&quot;abe49a09-78cc-4928-8309-d49f5e8f1043&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1602280740,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 6&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-6&quot;},&quot;pageUuid&quot;:&quot;abe49a09-78cc-4928-8309-d49f5e8f1043&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1602282347,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 6&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;933f6d1c-d33d-45a7-bf9b-c20080ff7cdf&quot;,&quot;whatsonId&quot;:&quot;10142736504831527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1602885540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1142,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:7,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 7&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-10-16-08h23m50s700-qia76o-qia76o.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=e6863992b97e66ec27fed9c4443733a4&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-7&quot;,&quot;midrollOffsets&quot;:[683],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14538&quot;,&quot;nodeUuid&quot;:&quot;7a9b66ae-fb67-475e-924d-5558987f89c3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1602885540,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 7&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-7&quot;},&quot;pageUuid&quot;:&quot;7a9b66ae-fb67-475e-924d-5558987f89c3&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1602886682,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 7&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;2753c1cf-81f2-4396-aa42-6fa6599d682c&quot;,&quot;whatsonId&quot;:&quot;10142736504840527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1603490340,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1236,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:8,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 8&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-10-23-08h15m53s849-qin5et-qin5et.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=534b0a4dae5baad8213482be2b96c2be&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-8&quot;,&quot;midrollOffsets&quot;:[639],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14539&quot;,&quot;nodeUuid&quot;:&quot;249fc4e8-b7a2-44e5-8c86-1838dbda1f7f&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1603490340,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 8&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-8&quot;},&quot;pageUuid&quot;:&quot;249fc4e8-b7a2-44e5-8c86-1838dbda1f7f&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1603491576,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 8&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;6b0a2a7b-484a-49e3-9df4-55b3598bac94&quot;,&quot;whatsonId&quot;:&quot;10142736504849527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1604098740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1228,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:9,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 9&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-10-30-12h11m47s117-qj0hsg-qj0hsg.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=cedec7a0cc147417ff84e9e0479f5cfa&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-9&quot;,&quot;midrollOffsets&quot;:[740],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14540&quot;,&quot;nodeUuid&quot;:&quot;aa9d2a10-6dfa-47d8-a7d1-5a557637200d&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1604098740,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 9&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-9&quot;},&quot;pageUuid&quot;:&quot;aa9d2a10-6dfa-47d8-a7d1-5a557637200d&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1604099968,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 9&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;17c7e542-46e7-4673-bc45-e8f9a20d86ff&quot;,&quot;whatsonId&quot;:&quot;10142736504858527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1604703540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1449,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:10,&quot;episodeTitle&quot;:&quot;S4 - Aflevering 10&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2020-11-06-09h10m07s146-qjd81x-qjd81x.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=0f074536054f854012f1f06674e17d51&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-10&quot;,&quot;midrollOffsets&quot;:[678],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14541&quot;,&quot;nodeUuid&quot;:&quot;3a3804c0-61dc-4e76-aefb-4969a18125f3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1604703540,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 10&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1911765540,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4\/wtfock-s4-aflevering-10&quot;},&quot;pageUuid&quot;:&quot;3a3804c0-61dc-4e76-aefb-4969a18125f3&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:4,&quot;seekableFrom&quot;:1604704989,&quot;title&quot;:&quot;wtFOCK - S4 - Aflevering 10&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S4 - Aflevering 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1911765540,&quot;videoUuid&quot;:&quot;3726d80a-4017-49b7-ac4d-9de4bae14a30&quot;,&quot;whatsonId&quot;:&quot;10142736504867527&quot;}],&quot;id&quot;:&quot;4f807a0c-b4d7-4db2-ba4c-1bd32c808761&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s4&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;14546&quot;,&quot;nodeUuid&quot;:&quot;4f807a0c-b4d7-4db2-ba4c-1bd32c808761&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1599145381,&quot;title&quot;:&quot;wtFOCK (najaar 2020)&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s4&quot;},&quot;title&quot;:&quot;Seizoen 4 - Kato&quot;},{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1571435940,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1826,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:1,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 1&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/1410wtfock-2-pzkfc2-pzkfc2.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=7d720e616d2a273ab827d4b8d5c9997e&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-23&quot;,&quot;midrollOffsets&quot;:[915],&quot;needs16PlusLabel&quot;:true,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14522&quot;,&quot;nodeUuid&quot;:&quot;0786b1eb-bce6-46c7-b13f-99cf55c1bcb1&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1571435940,&quot;title&quot;:&quot;Aflevering 23&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-23&quot;},&quot;pageUuid&quot;:&quot;0786b1eb-bce6-46c7-b13f-99cf55c1bcb1&quot;,&quot;parentalRating&quot;:&quot;16+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1571437766,&quot;title&quot;:&quot;Aflevering 23&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;461c9231-2aa8-4dc4-82c3-e18050432dac&quot;,&quot;whatsonId&quot;:&quot;10140645428841527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1572040740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1346,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:2,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 2&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/2210wtfockep24-3-pzxg36-pzxg36.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=348c1e07d8fad6c77a65ef2fd6987a6b&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-24&quot;,&quot;midrollOffsets&quot;:[635],&quot;needs16PlusLabel&quot;:true,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14523&quot;,&quot;nodeUuid&quot;:&quot;448c6104-0609-404b-af64-5ad5161ab6ea&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1572040740,&quot;title&quot;:&quot;Aflevering 24&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-24&quot;},&quot;pageUuid&quot;:&quot;448c6104-0609-404b-af64-5ad5161ab6ea&quot;,&quot;parentalRating&quot;:&quot;16+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1572042086,&quot;title&quot;:&quot;Aflevering 24&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;f3d9c296-06c2-4f8e-97eb-1ee157db502a&quot;,&quot;whatsonId&quot;:&quot;10140645429070527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1572894000,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1871,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:3,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 3&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/2610wtfock25-12-q0g0gj-q0g0gj.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=677075f287f5e9a5355c0b4e48e5bc65&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-25&quot;,&quot;midrollOffsets&quot;:[890],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14524&quot;,&quot;nodeUuid&quot;:&quot;8cbc4932-10de-4115-9af4-b83abb0b7be0&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1572894000,&quot;title&quot;:&quot;Aflevering 25&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-25&quot;},&quot;pageUuid&quot;:&quot;8cbc4932-10de-4115-9af4-b83abb0b7be0&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1572895871,&quot;title&quot;:&quot;Aflevering 25&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;0e8a6ac7-ef5e-4611-ab12-cb3d6eeff19c&quot;,&quot;whatsonId&quot;:&quot;10140645429079527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1573344000,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1973,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:4,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 4&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/0611wtfock26-55-q0ns65-q0ns65.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=864b6558bbb08d6e7c9330d4a50ba853&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-26&quot;,&quot;midrollOffsets&quot;:[995],&quot;needs16PlusLabel&quot;:true,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14525&quot;,&quot;nodeUuid&quot;:&quot;c3396224-57a6-4484-a1a4-ab7e63d75bbc&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1573344000,&quot;title&quot;:&quot;Aflevering 26&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308349,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-26&quot;},&quot;pageUuid&quot;:&quot;c3396224-57a6-4484-a1a4-ab7e63d75bbc&quot;,&quot;parentalRating&quot;:&quot;16+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1573345973,&quot;title&quot;:&quot;Aflevering 26&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308349,&quot;videoUuid&quot;:&quot;f918d18f-6709-4fdf-b465-8b232be578ef&quot;,&quot;whatsonId&quot;:&quot;10140645429088527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1573858740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:2273,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:5,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 5&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/wtfocky1e27-15-q1045d-q1045d.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=f32fc15e4134407e9fe69ed996a1ad2f&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-27&quot;,&quot;midrollOffsets&quot;:[1006],&quot;needs16PlusLabel&quot;:true,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14526&quot;,&quot;nodeUuid&quot;:&quot;5dd009c9-493a-4ac1-b58b-8a26fc3194f5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1573858740,&quot;title&quot;:&quot;Aflevering 27&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar-2019-aflevering-27&quot;},&quot;pageUuid&quot;:&quot;5dd009c9-493a-4ac1-b58b-8a26fc3194f5&quot;,&quot;parentalRating&quot;:&quot;16+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1573861013,&quot;title&quot;:&quot;Aflevering 27&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;01507a92-805d-478f-8401-dc40a434a5b0&quot;,&quot;whatsonId&quot;:&quot;10140645429097527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1574463540,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1580,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:6,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 6&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/wtfocky1e28-32-q1dcr5-q1dcr5.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=299dbf2528cf1312f2391e0f4e75de19&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-6&quot;,&quot;midrollOffsets&quot;:[861],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14527&quot;,&quot;nodeUuid&quot;:&quot;40ea2836-8242-4a62-9a7d-861e278b7b4a&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1574463540,&quot;title&quot;:&quot;Aflevering 28&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-6&quot;},&quot;pageUuid&quot;:&quot;40ea2836-8242-4a62-9a7d-861e278b7b4a&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1574465120,&quot;title&quot;:&quot;Aflevering 28&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;d096994c-3e96-474c-be3b-9d228aa38a1a&quot;,&quot;whatsonId&quot;:&quot;10140645429106527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1575154740,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1646,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:7,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 7&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/sputnikmedia-67f8e400023616still029-q1q3qy-q1q3qy.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=6ed2aad1f61bfd243f9f21085d7335e7&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-najaar2019-aflevering-29&quot;,&quot;midrollOffsets&quot;:[728],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14528&quot;,&quot;nodeUuid&quot;:&quot;b4bd4722-1f3d-4b69-9fb6-c5d858212ca0&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1575154740,&quot;title&quot;:&quot;Aflevering 29&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-najaar2019-aflevering-29&quot;},&quot;pageUuid&quot;:&quot;b4bd4722-1f3d-4b69-9fb6-c5d858212ca0&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1575156386,&quot;title&quot;:&quot;Aflevering 29&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;4395f179-68cd-4756-8c36-b7b1e2eedebc&quot;,&quot;whatsonId&quot;:&quot;10140645429115527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1575673140,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1616,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:8,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 8&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/fock3afl0810ref00080816still022-q232y9-q232y9.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=b3b3e6e85f92618c745aca4460bef7e4&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-8&quot;,&quot;midrollOffsets&quot;:[728],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14529&quot;,&quot;nodeUuid&quot;:&quot;40762121-3ebd-460c-b6dd-7f741e306b79&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1575673140,&quot;title&quot;:&quot;Aflevering 30&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-8&quot;},&quot;pageUuid&quot;:&quot;40762121-3ebd-460c-b6dd-7f741e306b79&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1575674756,&quot;title&quot;:&quot;Aflevering 30&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;a803045d-0b68-41bd-a5c3-fd86baabb6e6&quot;,&quot;whatsonId&quot;:&quot;10140645429124527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1576277940,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1486,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:9,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 9&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2019-12-13-11h26m05s589-q2g4zp-q2g4zp.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=5c99ae65c32218b66abd7d05f92dafe3&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-9&quot;,&quot;midrollOffsets&quot;:[771],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14530&quot;,&quot;nodeUuid&quot;:&quot;e0698806-cc21-4ff3-859f-4f9b566a73a7&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1576277940,&quot;title&quot;:&quot;Aflevering 31&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-9&quot;},&quot;pageUuid&quot;:&quot;e0698806-cc21-4ff3-859f-4f9b566a73a7&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1576279426,&quot;title&quot;:&quot;Aflevering 31&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;a0dfea76-0191-4ad7-b04d-8f793e7cbcd5&quot;,&quot;whatsonId&quot;:&quot;10140645429133527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1576969140,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:2032,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:10,&quot;episodeTitle&quot;:&quot;S3 - Aflevering 10&quot;,&quot;hasProductPlacement&quot;:true,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2019-12-19-10h23m59s549-q2r65l-q2r65l.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c0035576bcf19b43b36cce2d0a0591dc&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-10&quot;,&quot;midrollOffsets&quot;:[1080],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14531&quot;,&quot;nodeUuid&quot;:&quot;940c9fcc-d1ef-4e5b-85e8-f7f82e01ee1c&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1576969140,&quot;title&quot;:&quot;Aflevering 32&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;],&quot;unpublishDate&quot;:1767308340,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3\/wtfock-s3-aflevering-10&quot;},&quot;pageUuid&quot;:&quot;940c9fcc-d1ef-4e5b-85e8-f7f82e01ee1c&quot;,&quot;parentalRating&quot;:&quot;12+&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:3,&quot;seekableFrom&quot;:1576971172,&quot;title&quot;:&quot;Aflevering 32&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S3 - Aflevering 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:1767308340,&quot;videoUuid&quot;:&quot;6d4a6ad2-7cc9-480a-9e39-8c19b932fc05&quot;,&quot;whatsonId&quot;:&quot;10140645429142527&quot;}],&quot;id&quot;:&quot;d47a6be3-c498-40ee-9720-a1757e113407&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s3&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Karen&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten.&quot;,&quot;nodeId&quot;:&quot;14544&quot;,&quot;nodeUuid&quot;:&quot;d47a6be3-c498-40ee-9720-a1757e113407&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1571222414,&quot;title&quot;:&quot;wtFOCK (Najaar 2019)&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s3&quot;},&quot;title&quot;:&quot;Seizoen 3 - Robbe&quot;},{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;Nieuwe afleveringen&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1556387400,&quot;description&quot;:&quot;&lt;p&gt;Amber is nog steeds niet over haar zware crush op Senne. Maar Senne heeft zijn zinnen op Zo\u00eb gezet en probeert bij haar een date te versieren.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1499,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:1,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 1&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/wtfockafl-13-9-pqkrrc-pqkrrc.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=bcb478760886ef99432f796ce44571a7&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2018-aflevering-13&quot;,&quot;midrollOffsets&quot;:[750],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Amber is nog steeds niet over haar zware crush op Senne. Maar Senne heeft zijn zinnen op Zo\u00eb gezet en probeert bij haar een date te versieren.\r\n&quot;,&quot;nodeId&quot;:&quot;14512&quot;,&quot;nodeUuid&quot;:&quot;6a8d6c2a-1e88-4921-8d2e-3408640ec82e&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1556387400,&quot;title&quot;:&quot;Aflevering 13&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;Volledige Afleveringen&quot;,&quot;wtFOCK&quot;,&quot;Voorjaar 2019&quot;,&quot;Nieuwe afleveringen&quot;,&quot;nieuw seizoen&quot;,&quot;wtFOCK seizoen 2&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2018-aflevering-13&quot;},&quot;pageUuid&quot;:&quot;6a8d6c2a-1e88-4921-8d2e-3408640ec82e&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1556388899,&quot;title&quot;:&quot;Aflevering 13&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;732ec6bf-d83f-4d22-8fda-13fab24312af&quot;,&quot;whatsonId&quot;:&quot;10138522901390527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;Nieuwe afleveringen&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1556991600,&quot;description&quot;:&quot;&lt;p&gt;Zo\u00eb kan niet ontsnappen aan een date met Senne, maar probeert het verborgen te houden voor haar vriendinnen.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:2033,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:2,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 2&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2019-05-16-om-150922-prllvs-prllvs.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=d0f36af81eb534f1499e12906ce28cfd&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-14&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Zo\u00eb kan niet ontsnappen aan een date met Senne, maar probeert het verborgen te houden voor haar vriendinnen.&quot;,&quot;nodeId&quot;:&quot;14513&quot;,&quot;nodeUuid&quot;:&quot;f3c56eb0-5fca-44b7-b1ec-67c15a7d6841&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1556991600,&quot;title&quot;:&quot;Aflevering 14&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;Volledige Afleveringen&quot;,&quot;wtFOCK&quot;,&quot;Voorjaar 2019&quot;,&quot;Nieuwe afleveringen&quot;,&quot;Seizoen 2&quot;,&quot;nieuw seizoen&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-14&quot;},&quot;pageUuid&quot;:&quot;f3c56eb0-5fca-44b7-b1ec-67c15a7d6841&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1556993633,&quot;title&quot;:&quot;Aflevering 14&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;2f12e027-e9c1-41de-b73e-e9c2d27518de&quot;,&quot;whatsonId&quot;:&quot;10138522901398527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1557596100,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1990,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:3,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 3&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2019-05-16-om-144749-prlkvv-prlkvv.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=877a8d37b3129878ebf99295729a47ef&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-15&quot;,&quot;midrollOffsets&quot;:[1060],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14514&quot;,&quot;nodeUuid&quot;:&quot;c7a2f7d5-9bd2-471e-93b3-f97e645d11c5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1557596100,&quot;title&quot;:&quot;Aflevering 15&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;Volledige Afleveringen&quot;,&quot;wtFOCK&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-15&quot;},&quot;pageUuid&quot;:&quot;c7a2f7d5-9bd2-471e-93b3-f97e645d11c5&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1557598090,&quot;title&quot;:&quot;Aflevering 15&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;5de464d7-fdc6-4aa9-929f-511f715eb835&quot;,&quot;whatsonId&quot;:&quot;10138522901406527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1558201500,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1709,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:4,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 4&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2019-05-15-om-104152-prnmva-prnmva.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=2f15e0da52480d2358f795af72f54130&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-16&quot;,&quot;midrollOffsets&quot;:[852],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14515&quot;,&quot;nodeUuid&quot;:&quot;34552311-6faf-4561-a888-7905c1b57471&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1558201500,&quot;title&quot;:&quot;Aflevering 16&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;Volledige Afleveringen&quot;,&quot;wtFOCK&quot;,&quot;Aflevering 16&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-16&quot;},&quot;pageUuid&quot;:&quot;34552311-6faf-4561-a888-7905c1b57471&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1558203209,&quot;title&quot;:&quot;Aflevering 16&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;c2db434f-84d9-4faa-98be-33f1f4df564c&quot;,&quot;whatsonId&quot;:&quot;10138522901414527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1558797300,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1517,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:5,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 5&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2019-05-22-om-145837-ps0nxw-ps0nxw.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c324529b86e8f3fe0ba752abd05f3495&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-17&quot;,&quot;midrollOffsets&quot;:[774],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14516&quot;,&quot;nodeUuid&quot;:&quot;fb888d42-b526-4c9b-a482-b25b917cee89&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1558797300,&quot;title&quot;:&quot;Aflevering 17&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Voorjaar 2019&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-17&quot;},&quot;pageUuid&quot;:&quot;fb888d42-b526-4c9b-a482-b25b917cee89&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1558798817,&quot;title&quot;:&quot;Aflevering 17&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;91a17dbe-a415-4dfa-9149-c531df1aa60c&quot;,&quot;whatsonId&quot;:&quot;10138522901422527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1559402053,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:2026,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:6,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 6&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/fock-2afl-060900032014still013-psdjw4-psdjw4.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=53a8cc072ee6ccf386d235efea10c8ea&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-18&quot;,&quot;midrollOffsets&quot;:[1137],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14517&quot;,&quot;nodeUuid&quot;:&quot;4ebaa517-d280-44ec-bdee-fd247d391249&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1559402053,&quot;title&quot;:&quot;Aflevering 18&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Voorjaar 2019&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-voorjaar-2019-aflevering-18&quot;},&quot;pageUuid&quot;:&quot;4ebaa517-d280-44ec-bdee-fd247d391249&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1559404079,&quot;title&quot;:&quot;Aflevering 18&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;5c1f2d1c-fc68-43f8-b113-28075c687cf3&quot;,&quot;whatsonId&quot;:&quot;10138522901430527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1560007200,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1788,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:7,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 7&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/wtfock191100032015still007-psqk66-psqk66.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=541fe1876778dcccd851db43402db3cd&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-7-08062019&quot;,&quot;midrollOffsets&quot;:[932],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14518&quot;,&quot;nodeUuid&quot;:&quot;2b15f52f-358c-4508-8578-d76e75f7b583&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1560007200,&quot;title&quot;:&quot;Aflevering 19&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Voorjaar 2019&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-7-08062019&quot;},&quot;pageUuid&quot;:&quot;2b15f52f-358c-4508-8578-d76e75f7b583&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1560008988,&quot;title&quot;:&quot;Aflevering 19&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;d95e8d29-43b2-40ae-89db-009f9b77aec5&quot;,&quot;whatsonId&quot;:&quot;10138522901438527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1560611400,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1868,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:8,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 8&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/wtfock20300073024still014-pt3jlz-pt3jlz.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=3cdc85345bc0cffcea8952b8523ad3bc&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-8-15062019&quot;,&quot;midrollOffsets&quot;:[850],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14519&quot;,&quot;nodeUuid&quot;:&quot;bde53e93-738c-44da-999b-220139e45132&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1560611400,&quot;title&quot;:&quot;Aflevering 20&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-8-15062019&quot;},&quot;pageUuid&quot;:&quot;bde53e93-738c-44da-999b-220139e45132&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1560613268,&quot;title&quot;:&quot;Aflevering 20&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;d1541cf3-045f-45ac-94fb-5bf3acb3b5ed&quot;,&quot;whatsonId&quot;:&quot;10138522901446527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1561216200,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1445,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:9,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 9&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2019-06-14-om-145241-ptgfoq-ptgfoq.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=707171b7252b279186b7d1bb9cd3d694&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-9-22062019&quot;,&quot;midrollOffsets&quot;:[790],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14520&quot;,&quot;nodeUuid&quot;:&quot;986f0dab-0717-4ee2-8e59-3f4af1f268ea&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1561216200,&quot;title&quot;:&quot;Aflevering 21&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-9-22062019&quot;},&quot;pageUuid&quot;:&quot;986f0dab-0717-4ee2-8e59-3f4af1f268ea&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1561217645,&quot;title&quot;:&quot;Aflevering 21&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;694a336f-672b-4408-b200-89d0f0b1a278&quot;,&quot;whatsonId&quot;:&quot;10138522901454527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1561821300,&quot;description&quot;:&quot;&lt;p&gt;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1804,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:10,&quot;episodeTitle&quot;:&quot;S2 - Aflevering 10&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/fock-2afl-1009-v3-1-zonder-ot00022317still006-ptyr2o-ptyr2o.jpg?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=b36f0f4625d050ae7ada6fcd30c171ad&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:false,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-22-29062019&quot;,&quot;midrollOffsets&quot;:[856],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Verliefdheid, vriendschap, nijd, taboes en onzekerheden van het tiener zijn spelen de hoofdrol in het vervolg van de meest succesvolle online reeks.&quot;,&quot;nodeId&quot;:&quot;14521&quot;,&quot;nodeUuid&quot;:&quot;bb58809e-9922-46f4-b181-9cd88225a4d0&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1561821300,&quot;title&quot;:&quot;Aflevering 22&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;,&quot;Volledige Afleveringen&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2\/wtfock-s2-aflevering-22-29062019&quot;},&quot;pageUuid&quot;:&quot;bb58809e-9922-46f4-b181-9cd88225a4d0&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:2,&quot;seekableFrom&quot;:1561823104,&quot;title&quot;:&quot;Aflevering 22&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S2 - Aflevering 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;2b96f2e2-318b-4096-87f9-49030ba4f73c&quot;,&quot;whatsonId&quot;:&quot;10138522901462527&quot;}],&quot;id&quot;:&quot;38fe9d7d-5cf8-45fd-8e5d-c5dab97f9d0b&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s2&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten.&quot;,&quot;nodeId&quot;:&quot;14543&quot;,&quot;nodeUuid&quot;:&quot;38fe9d7d-5cf8-45fd-8e5d-c5dab97f9d0b&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554194925,&quot;title&quot;:&quot;wtFOCK (voorjaar 2019)&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s2&quot;},&quot;title&quot;:&quot;Seizoen 2 - Zo\u00eb&quot;},{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;&quot;,&quot;createdDate&quot;:1545458401,&quot;description&quot;:&quot;&lt;p&gt;De 15-jarige Jana is nog maar net op de nieuwe school waar haar vriendje Jens ook zit. Om nieuwe vriendinnen te maken, gaat ze naar het Kick Off-feestje van de school, maar daar komt het tot een confrontatie met Britt, die haar een slet noemt. Zo\u00eb, die ook nieuw is in de school, heeft het gezien en spreekt Jana troostend aan. De twee kunnen het best met elkaar vinden.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1413,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:1,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 1&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/schermafbeelding-2018-12-10-om-114025-pk34kw-pk34kw.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=96258820dcfbe05f0fb0033d26480eab&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/aflevering-1&quot;,&quot;midrollOffsets&quot;:[837,837],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Axel.Florizoone@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;De 15-jarige Jana is nog maar net op de nieuwe school waar haar vriendje Jens ook zit. &quot;,&quot;nodeId&quot;:&quot;13317&quot;,&quot;nodeUuid&quot;:&quot;b9cee7f0-32a1-431f-99b2-504e36109c78&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1545458401,&quot;title&quot;:&quot;wtFOCK -Aflevering 1&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Aflevering&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmina&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;senne&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Moyo&quot;,&quot;kerstfictie&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/aflevering-1&quot;},&quot;pageUuid&quot;:&quot;b9cee7f0-32a1-431f-99b2-504e36109c78&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;wtFOCK -Aflevering 1&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;92c7253e-8ea0-41bf-8b49-3b6e9bef9072&quot;,&quot;whatsonId&quot;:&quot;10137137691310527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Jana heeft een romantisch weekendje Ardennen geregeld met Jens, maar Jens wil zijn beste vriend Robbe ook meenemen. Jana is teleurgesteld en het weekendje-uit loopt anders dan ze had gehoopt.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1282,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:2,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 2&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl2-ppbtlb-ppbtlb.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=d53ab4889c912799f124cddccd4adfc0&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-2&quot;,&quot;midrollOffsets&quot;:[675],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Jana heeft een romantisch weekendje Ardennen geregeld met Jens, maar Jens wil zijn beste vriend Robbe ook meenemen. Jana is teleurgesteld en het weekendje-uit loopt anders dan ze had gehoopt.&quot;,&quot;nodeId&quot;:&quot;14510&quot;,&quot;nodeUuid&quot;:&quot;7c31d268-f168-4ae1-800b-dac89968eafd&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 2&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-2&quot;},&quot;pageUuid&quot;:&quot;7c31d268-f168-4ae1-800b-dac89968eafd&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 2&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;91418946-ae94-4e1e-ad24-e4153c0868e0&quot;,&quot;whatsonId&quot;:&quot;10138327665171527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Jana leert Zo\u00eb beter kennen en samen met klasgenoten Luca en Amber vormen ze een team om mee te doen met de Freefest-wedstrijd op school.&amp;nbsp;&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1077,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:3,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 3&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl3-ppbsyh-ppbsyh.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=d83cb13b8d1bd142dd614361437c23d5&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-3&quot;,&quot;midrollOffsets&quot;:[521],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Jana leert Zo\u00eb beter kennen en samen met klasgenoten Luca en Amber vormen ze een team om mee te doen met de Freefest-wedstrijd op school. &quot;,&quot;nodeId&quot;:&quot;14500&quot;,&quot;nodeUuid&quot;:&quot;6134f6ad-58c2-4b2e-abca-5a6731dccaf3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 3&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-3&quot;},&quot;pageUuid&quot;:&quot;6134f6ad-58c2-4b2e-abca-5a6731dccaf3&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 3&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;78295b88-c90e-4cdf-8278-6bc259a9250d&quot;,&quot;whatsonId&quot;:&quot;10138327665171527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;De meisjes besluiten aansluiting te zoeken met de coolste boys van de school om hun kansen tijdens de Freefest-wedstrijd te vergroten.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1190,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:4,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 4&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl4-ppbt0f-ppbt0f.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=5edb1ad24f3544805c10707ea7475fe5&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-4&quot;,&quot;midrollOffsets&quot;:[521],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;De meisjes besluiten aansluiting te zoeken met de coolste boys van de school om hun kansen tijdens de Freefest-wedstrijd te vergroten.&quot;,&quot;nodeId&quot;:&quot;14501&quot;,&quot;nodeUuid&quot;:&quot;594490c1-aeb6-4fa7-b0bc-b6137e9f295f&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 4&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-4&quot;},&quot;pageUuid&quot;:&quot;594490c1-aeb6-4fa7-b0bc-b6137e9f295f&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 4&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;807ee8dc-bbb2-4963-af9c-feb02e93005f&quot;,&quot;whatsonId&quot;:&quot;10138327665177527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Jana vermoedt dat Jens zaken voor haar verbergt en dat hij contact heeft met Britt.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1429,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:5,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 5&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl5-ppbt22-ppbt22.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c01d6b8885582b7c91ee9989e72066b4&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-5&quot;,&quot;midrollOffsets&quot;:[602],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Jana vermoedt dat Jens zaken voor haar verbergt en dat hij contact heeft met Britt.&quot;,&quot;nodeId&quot;:&quot;14502&quot;,&quot;nodeUuid&quot;:&quot;a8955147-8294-49bc-af31-2c6cd02ebbe9&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 5&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-5&quot;},&quot;pageUuid&quot;:&quot;a8955147-8294-49bc-af31-2c6cd02ebbe9&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 5&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;3e831fc9-f599-494d-9d30-dd0152d5ab81&quot;,&quot;whatsonId&quot;:&quot;10138327665177527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Amber loopt op wolken na haar nacht met Senne, maar niet iedereen deelt haar enthousiasme.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1272,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:6,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 6&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl6-ppbta1-ppbta1.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=3b45d38e1169c17c16347befe3b0b19b&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-6&quot;,&quot;midrollOffsets&quot;:[535],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Amber loopt op wolken na haar nacht met Senne, maar niet iedereen deelt haar enthousiasme.&quot;,&quot;nodeId&quot;:&quot;14504&quot;,&quot;nodeUuid&quot;:&quot;2d0f371b-b23b-471b-9988-9f91b9c3e740&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 6&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-6&quot;},&quot;pageUuid&quot;:&quot;2d0f371b-b23b-471b-9988-9f91b9c3e740&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 6&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;a0a06ad8-42fc-4c53-b1ae-d8ac11d5a80b&quot;,&quot;whatsonId&quot;:&quot;10138327665183527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Het gaat weer beter tussen Jana en Jens, maar Jana draagt een geheim met zich mee en neemt Robbe in vertrouwen.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1230,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:7,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 7&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl7-ppbt6b-ppbt6b.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=b131f6f32ad88c8f1ed8a2b3675bcaa0&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-7&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Het gaat weer beter tussen Jana en Jens, maar Jana draagt een geheim met zich mee en neemt Robbe in vertrouwen.&quot;,&quot;nodeId&quot;:&quot;14503&quot;,&quot;nodeUuid&quot;:&quot;638cb4d2-0243-4bbb-826c-eb925a633fd4&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 7&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-7&quot;},&quot;pageUuid&quot;:&quot;638cb4d2-0243-4bbb-826c-eb925a633fd4&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 7&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;4ef2ac18-6061-4e25-8f2f-28e3d7df98da&quot;,&quot;whatsonId&quot;:&quot;10138327665183527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Wanneer bekend raakt dat Jana op een feestje met een andere jongen heeft gekust, wil Jens haar niet meer zien.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1237,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:8,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 8&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl8-ppbtcf-ppbtcf.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=a2f889f813292e8aa0c91abaa0367692&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-8&quot;,&quot;midrollOffsets&quot;:[640],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Wanneer bekend raakt dat Jana op een feestje met een andere jongen heeft gekust, wil Jens haar niet meer zien.&quot;,&quot;nodeId&quot;:&quot;14505&quot;,&quot;nodeUuid&quot;:&quot;c9e19584-3f79-4da4-91f1-5ec309fd0e06&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 8&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-8&quot;},&quot;pageUuid&quot;:&quot;c9e19584-3f79-4da4-91f1-5ec309fd0e06&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 8&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;10f08fde-c24d-4688-891f-14ffd6f9814f&quot;,&quot;whatsonId&quot;:&quot;10138327665183527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Jana probeert het goed te maken met Jens, maar Jens antwoordt dat Jana eerst moet uitzoeken wat voor iemand ze wil zijn.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1399,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:9,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 9&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl9-ppbted-ppbted.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=aca981918b3273f36ac17a6cf37b6c7e&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-9&quot;,&quot;midrollOffsets&quot;:[718],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Jana probeert het goed te maken met Jens, maar Jens antwoordt dat Jana eerst moet uitzoeken wat voor iemand ze wil zijn.&quot;,&quot;nodeId&quot;:&quot;14506&quot;,&quot;nodeUuid&quot;:&quot;7739464a-2fec-48db-9d6f-eea935f22327&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 9&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-9&quot;},&quot;pageUuid&quot;:&quot;7739464a-2fec-48db-9d6f-eea935f22327&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166326,&quot;title&quot;:&quot;Aflevering 9&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;a2201e09-a522-4276-9bc6-a6d0b8b526b7&quot;,&quot;whatsonId&quot;:&quot;10138327665189527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Amber wordt onwel op het feestje van Marie.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1314,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:10,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 10&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl10-ppbtg7-ppbtg7.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=724e4f64abb362c616a5c8f7c122ffb7&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-10&quot;,&quot;midrollOffsets&quot;:[648],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Amber wordt onwel op het feestje van Marie.&quot;,&quot;nodeId&quot;:&quot;14507&quot;,&quot;nodeUuid&quot;:&quot;557eaf64-2048-4f24-a77b-733b3c754dad&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 10&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-10&quot;},&quot;pageUuid&quot;:&quot;557eaf64-2048-4f24-a77b-733b3c754dad&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166327,&quot;title&quot;:&quot;Aflevering 10&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;b8328b65-fa73-4704-b7ea-49845059939d&quot;,&quot;whatsonId&quot;:&quot;10138327665189527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Amber vreest dat ze zwanger is van die ene keer seks met Senne en vraagt haar vriendinnen om mee te gaan naar de dokter.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1074,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:11,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 11&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl11-ppbthw-ppbthw.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=d048a7ecfe2d018e3cb136ff75a9d9fc&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-11&quot;,&quot;midrollOffsets&quot;:[622],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Amber vreest dat ze zwanger is van die ene keer seks met Senne en vraagt haar vriendinnen om mee te gaan naar de dokter.&quot;,&quot;nodeId&quot;:&quot;14508&quot;,&quot;nodeUuid&quot;:&quot;bc80c5ee-0b4f-47dd-896c-3bca544dd901&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 11&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-11&quot;},&quot;pageUuid&quot;:&quot;bc80c5ee-0b4f-47dd-896c-3bca544dd901&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166327,&quot;title&quot;:&quot;Aflevering 11&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 11&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;59ed87c2-0dc1-46bc-b706-b16632b57bb1&quot;,&quot;whatsonId&quot;:&quot;10138327665195527&quot;},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.tvi.ep.vod.free&quot;,&quot;createdDate&quot;:1554568800,&quot;description&quot;:&quot;&lt;p&gt;Jana weet ondertussen wie haar verraden heeft en op de kerstdrink op school wordt ook duidelijk waarom.&lt;\/p&gt;\r\n&quot;,&quot;duration&quot;:1612,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:12,&quot;episodeTitle&quot;:&quot;S1 - Aflevering 12&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/afl12-ppbtjk-ppbtjk.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=2fe468e70f7da35114ec9c98e2d580b8&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-12&quot;,&quot;midrollOffsets&quot;:[823],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;Jana weet ondertussen wie haar verraden heeft en op de kerstdrink op school wordt ook duidelijk waarom.&quot;,&quot;nodeId&quot;:&quot;14509&quot;,&quot;nodeUuid&quot;:&quot;6f70e681-bda3-4174-8e85-4488c91336eb&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554568800,&quot;title&quot;:&quot;Aflevering 12&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[&quot;wtFOCK&quot;,&quot;Volledige Afleveringen&quot;,&quot;Seizoen 1&quot;,&quot;Jana&quot;,&quot;Jens&quot;,&quot;Robbe&quot;,&quot;Amber&quot;,&quot;Zo\u00eb&quot;,&quot;Yasmine&quot;,&quot;Luca&quot;,&quot;Luka&quot;,&quot;Britt&quot;,&quot;Marie&quot;,&quot;Moyo&quot;],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1\/wtfock-seizoen-1-aflevering-12&quot;},&quot;pageUuid&quot;:&quot;6f70e681-bda3-4174-8e85-4488c91336eb&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=crop&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=54c9efbf04b07473c346079b23123344&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:1,&quot;seekableFrom&quot;:1626166327,&quot;title&quot;:&quot;Aflevering 12&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;S1 - Aflevering 12&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;d7230a6e-08f2-44d3-b29a-ea6c5a197719&quot;,&quot;whatsonId&quot;:&quot;10138327665195527&quot;}],&quot;id&quot;:&quot;ddf5ff86-e2aa-4cfd-b117-72f9b56add63&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfock-s1&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;Sara&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten.&quot;,&quot;nodeId&quot;:&quot;14542&quot;,&quot;nodeUuid&quot;:&quot;ddf5ff86-e2aa-4cfd-b117-72f9b56add63&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1554194925,&quot;title&quot;:&quot;wtFOCK (najaar 2018)&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfock-s1&quot;},&quot;title&quot;:&quot;Seizoen 1 - Jana&quot;},{&quot;episodes&quot;:[{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955702,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:177,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 32&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/32-qy0vxs.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c9fd580dfb20d1828af868715e6993c6&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-32&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19776&quot;,&quot;nodeUuid&quot;:&quot;371e8f39-79c3-4267-b350-f18fba9986b3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955702,&quot;title&quot;:&quot;WtfockDown - Afl 32&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-32&quot;},&quot;pageUuid&quot;:&quot;371e8f39-79c3-4267-b350-f18fba9986b3&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 32&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 32&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;0955ea8d-4045-4fc2-8895-15580339f12a&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955657,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:216,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 31&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/31-qy0vwj.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=d984e97615fb82ecb55f9c9c968bdf24&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-31&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19775&quot;,&quot;nodeUuid&quot;:&quot;cfa6ec37-88f9-4fb3-94bd-71d48ef015a2&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955657,&quot;title&quot;:&quot;WtfockDown - Afl 31&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-31&quot;},&quot;pageUuid&quot;:&quot;cfa6ec37-88f9-4fb3-94bd-71d48ef015a2&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 31&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 31&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;fe10b268-881a-48c5-80ce-f15e0b70848a&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955603,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:146,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 30&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/30-qy0vv2.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=26e051a59ecf577137b168ec792a3487&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-30&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19774&quot;,&quot;nodeUuid&quot;:&quot;9748e4f3-d00f-47fd-8e50-483735957ef1&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955603,&quot;title&quot;:&quot;WtfockDown - Afl 30&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-30&quot;},&quot;pageUuid&quot;:&quot;9748e4f3-d00f-47fd-8e50-483735957ef1&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 30&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 30&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;116b3c49-3fd5-4126-98bc-9f2a9f43f90a&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955554,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:122,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 29&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/29-qy0vts.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=16c58b6aa008d4a5b98f0bdf47fbc304&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-29&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19773&quot;,&quot;nodeUuid&quot;:&quot;c20f40a4-0955-4483-8d3b-c19f2786f54f&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955554,&quot;title&quot;:&quot;WtfockDown - Afl 29&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-29&quot;},&quot;pageUuid&quot;:&quot;c20f40a4-0955-4483-8d3b-c19f2786f54f&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 29&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 29&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;e28b6d82-91fb-42cd-9a81-c72a106dcf0b&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955507,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:89,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 28&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/28-qy0vse.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=0bf465cea72eae32e77d474c77cd3033&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-28&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19772&quot;,&quot;nodeUuid&quot;:&quot;936cf085-2cb8-4a37-ad7c-a2172096de4a&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955507,&quot;title&quot;:&quot;WtfockDown - Afl 28&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-28&quot;},&quot;pageUuid&quot;:&quot;936cf085-2cb8-4a37-ad7c-a2172096de4a&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 28&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 28&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;ccf61fb5-eb26-4ff4-a168-740fd715bedc&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955456,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:108,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 27&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/27-qy0vqy.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=ffda3c57cca88c9c8326c5b5f8da4712&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-27&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19771&quot;,&quot;nodeUuid&quot;:&quot;bd1637ea-42e9-4e39-9b9b-9b2bf69f1fa4&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955456,&quot;title&quot;:&quot;WtfockDown - Afl 27&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-27&quot;},&quot;pageUuid&quot;:&quot;bd1637ea-42e9-4e39-9b9b-9b2bf69f1fa4&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 27&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 27&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;56f85de6-b77e-41b2-901e-422f2cad6fa9&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955406,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:115,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 26&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/26-qy0vpr.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=f7026b09a1c2d1fd954389dd9f63d341&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-26&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19770&quot;,&quot;nodeUuid&quot;:&quot;7a7b8188-3a8b-4dba-965a-188ac0a5baf5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955406,&quot;title&quot;:&quot;WtfockDown - Afl 26&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-26&quot;},&quot;pageUuid&quot;:&quot;7a7b8188-3a8b-4dba-965a-188ac0a5baf5&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 26&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 26&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;6b09d6be-6e4a-4bc8-b201-42c7f1c83f16&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955329,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:112,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 25&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/25-qy0vng.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=95120e1170612f41ff2f74e419239d74&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-25&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19769&quot;,&quot;nodeUuid&quot;:&quot;3595494e-9808-4261-86de-b1f9498de786&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955329,&quot;title&quot;:&quot;WtfockDown - Afl 25&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-25&quot;},&quot;pageUuid&quot;:&quot;3595494e-9808-4261-86de-b1f9498de786&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 25&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 25&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;ddb4eb2d-ab28-4bce-90ca-1ceb48c4cb3e&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955284,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:152,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 24&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/24-qy0vm7.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=a1ae7c0b02c3ce3d1106f25eeac891ed&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-24&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19768&quot;,&quot;nodeUuid&quot;:&quot;91f94856-b4c7-446e-b9bb-642ae3cc941c&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955284,&quot;title&quot;:&quot;WtfockDown - Afl 24&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-24&quot;},&quot;pageUuid&quot;:&quot;91f94856-b4c7-446e-b9bb-642ae3cc941c&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 24&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 24&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;7cc3dc62-ef84-4016-a6a0-dbcd5af98451&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955236,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:126,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 23&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/23-qy0vkv.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=cb362828339bf2bee46a6c5d75e6f9ea&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-23&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19767&quot;,&quot;nodeUuid&quot;:&quot;e3c83b76-d0dc-435c-bc47-db933f63ff5e&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955236,&quot;title&quot;:&quot;WtfockDown - Afl 23&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-23&quot;},&quot;pageUuid&quot;:&quot;e3c83b76-d0dc-435c-bc47-db933f63ff5e&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 23&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 23&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;4db68da3-5b08-4169-8802-6514ff4bdccf&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955184,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:77,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 22&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/22-qy0vje.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=97ded0b3fd7b79340f82434b06cbf6c4&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-22&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19766&quot;,&quot;nodeUuid&quot;:&quot;9053903c-fe12-4d66-b383-d1cd59ac2064&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955184,&quot;title&quot;:&quot;WtfockDown - Afl 22&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-22&quot;},&quot;pageUuid&quot;:&quot;9053903c-fe12-4d66-b383-d1cd59ac2064&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 22&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 22&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;620c8e1a-9053-452e-9c9c-2037bd8741de&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610955092,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:128,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 21&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/21-qy0vgv.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=e50d531b1cde558d076dfff266ec04af&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-21&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19765&quot;,&quot;nodeUuid&quot;:&quot;4c4e658f-6fe6-487b-adce-56a1157699ec&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610955092,&quot;title&quot;:&quot;WtfockDown - Afl 21&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-21&quot;},&quot;pageUuid&quot;:&quot;4c4e658f-6fe6-487b-adce-56a1157699ec&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 21&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 21&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;38408c2c-78d4-4ae0-808b-6ae083e24c5a&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954985,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:156,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 20&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/20-qy0vdz.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=e83d1a716c223b420c2458ccaacf8b25&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-20&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19764&quot;,&quot;nodeUuid&quot;:&quot;a48cda8e-61ab-468e-8d20-9e20f48ccdae&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954985,&quot;title&quot;:&quot;WtfockDown - Afl 20&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-20&quot;},&quot;pageUuid&quot;:&quot;a48cda8e-61ab-468e-8d20-9e20f48ccdae&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 20&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 20&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;f60cb6bf-1dfb-4774-bac3-4183ea3fcf5b&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954932,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:167,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 19&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/19-qy0vce.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=3d83c259bbc724447be9202b4bfc79da&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-19&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19763&quot;,&quot;nodeUuid&quot;:&quot;b820502a-c960-4a8a-946b-76e90592b5b3&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954932,&quot;title&quot;:&quot;WtfockDown - Afl 19&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-19&quot;},&quot;pageUuid&quot;:&quot;b820502a-c960-4a8a-946b-76e90592b5b3&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 19&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 19&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;0eb6763a-f893-40f4-8f44-2207cbd0c729&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954881,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:131,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 18&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/18-qy0vb1.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c400188f466fcee70df74d99e9983697&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-18&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19762&quot;,&quot;nodeUuid&quot;:&quot;69debd53-fff3-4792-b586-fb45e473ca73&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954881,&quot;title&quot;:&quot;WtfockDown - Afl 18&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-18&quot;},&quot;pageUuid&quot;:&quot;69debd53-fff3-4792-b586-fb45e473ca73&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 18&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 18&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;020247cf-fc8f-45be-b51f-1371b27651de&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954773,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:149,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 17&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/17-qy0v7z.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=e6d1c607e16cea371af5672a8f1b07ee&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-17&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19761&quot;,&quot;nodeUuid&quot;:&quot;9c7f716e-733a-4f05-8c45-a0a0bf97e6c5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954773,&quot;title&quot;:&quot;WtfockDown - Afl 17&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-17&quot;},&quot;pageUuid&quot;:&quot;9c7f716e-733a-4f05-8c45-a0a0bf97e6c5&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 17&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 17&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;087d670f-e386-4e83-9ddd-83e201a762ae&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954720,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:105,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 16&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/16-qy0v6l.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=b4a8bb8acc2a1ab06de754812e5fd022&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-16&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19760&quot;,&quot;nodeUuid&quot;:&quot;25f38ab5-f6a3-41e1-a6df-b891b5c2600b&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954720,&quot;title&quot;:&quot;WtfockDown - Afl 16&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-16&quot;},&quot;pageUuid&quot;:&quot;25f38ab5-f6a3-41e1-a6df-b891b5c2600b&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 16&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 16&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;60e985d3-2b3c-478e-a1d4-ab5e7efd677e&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954630,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:103,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 15&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/15-qy0v4b-qz66cr.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=3b23b780dccd1dddf50526457f28a6f3&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-15&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19759&quot;,&quot;nodeUuid&quot;:&quot;9b679100-e74b-4de9-8565-32d66edf59f9&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954630,&quot;title&quot;:&quot;WtfockDown - Afl 15&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-15&quot;},&quot;pageUuid&quot;:&quot;9b679100-e74b-4de9-8565-32d66edf59f9&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 15&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 15&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;45f0a067-e154-4b02-8ecd-ac75ad2562b4&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954587,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:223,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 14&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/14-qy0v2u.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c1f69c0f7d6c5b13914423429b0138f8&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-14&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19758&quot;,&quot;nodeUuid&quot;:&quot;5207954f-e267-4da0-a604-1f427a0701a2&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954587,&quot;title&quot;:&quot;WtfockDown - Afl 14&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-14&quot;},&quot;pageUuid&quot;:&quot;5207954f-e267-4da0-a604-1f427a0701a2&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 14&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 14&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;6155707b-9d9b-413a-af85-ca8ae2894512&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954544,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:180,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 13&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/13-qy0v1m.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c88c880474866eecda197b28af65a0de&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-13&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19757&quot;,&quot;nodeUuid&quot;:&quot;45bebe21-4a1b-4d8c-a0b6-c4f26f0e50ee&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954544,&quot;title&quot;:&quot;WtfockDown - Afl 13&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-13&quot;},&quot;pageUuid&quot;:&quot;45bebe21-4a1b-4d8c-a0b6-c4f26f0e50ee&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 13&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 13&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;5963f934-f727-4aa4-bbf5-a4f256285b2e&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954497,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:90,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 12&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/12-qy0v0b.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=98b8105a372e7c2b1ad477980e236fbd&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-12&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19756&quot;,&quot;nodeUuid&quot;:&quot;a4417760-c972-44b3-8f45-a05d191a0aae&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954497,&quot;title&quot;:&quot;WtfockDown - Afl 12&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-12&quot;},&quot;pageUuid&quot;:&quot;a4417760-c972-44b3-8f45-a05d191a0aae&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 12&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 12&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;6ecaa1a9-48ca-42b5-a3c9-4b8d5907798c&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610954378,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:210,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 11&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/11-qy0uyd.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=42f7ee12eb5d4adb9fc786598be30b79&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-11&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19755&quot;,&quot;nodeUuid&quot;:&quot;ae3550fd-855a-4d12-a342-29e08ed1898b&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610954378,&quot;title&quot;:&quot;WtfockDown - Afl 11&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-11&quot;},&quot;pageUuid&quot;:&quot;ae3550fd-855a-4d12-a342-29e08ed1898b&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 11&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 11&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;44e68660-4ba6-40c1-bd36-09ebc5133c7f&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902167,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:89,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 10&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/10-qxzqp7.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=eb57b3128f66003681c6a21978218b82&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-10&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19741&quot;,&quot;nodeUuid&quot;:&quot;64b5bfae-8a4b-41de-bb23-4b355ee87bc5&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902167,&quot;title&quot;:&quot;Wtfockdown - Afl 10&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-10&quot;},&quot;pageUuid&quot;:&quot;64b5bfae-8a4b-41de-bb23-4b355ee87bc5&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 10&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 10&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;74dbf6cd-a7bd-4bed-a33a-2d76927740e5&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610903232,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:125,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 9&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/09-qxzr7b.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=5b616bddfb816a8495eab23e682889b1&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-9&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19749&quot;,&quot;nodeUuid&quot;:&quot;03befebb-9daa-4871-9cff-a47a94c6c383&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610903232,&quot;title&quot;:&quot;Wtfockdown - Afl 9&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-9&quot;},&quot;pageUuid&quot;:&quot;03befebb-9daa-4871-9cff-a47a94c6c383&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 9&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 9&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;392c4c24-6f51-4720-8b28-2808faa936de&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902162,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:233,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 8&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/08-qxzqp1.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=986fe703593a11a7fbfc92e3a2821a70&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-8&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19742&quot;,&quot;nodeUuid&quot;:&quot;cdf8c2f8-5635-440d-b36a-f73822349c33&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902162,&quot;title&quot;:&quot;Wtfockdown - Afl 8&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-8&quot;},&quot;pageUuid&quot;:&quot;cdf8c2f8-5635-440d-b36a-f73822349c33&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 8&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 8&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;5268a93b-de87-4dc4-bdad-543e62ff5ba0&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902852,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:159,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 7&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/07-qxzr5t.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=aba62160e0c718b7bd7a8456cef474c0&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-7&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19748&quot;,&quot;nodeUuid&quot;:&quot;a3114d2e-4851-435e-a851-45c279ce191d&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902852,&quot;title&quot;:&quot;Wtfockdown - Afl 7&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-7&quot;},&quot;pageUuid&quot;:&quot;a3114d2e-4851-435e-a851-45c279ce191d&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 7&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 7&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;88fcf309-dfd7-4bcf-9426-15360e2b16a4&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902157,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:200,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 6&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/06-qxzqot.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=737518ceb6adaebd9ac44e69368510fe&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-6&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19743&quot;,&quot;nodeUuid&quot;:&quot;792d2866-fb20-4f33-b77c-56142384f7d2&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902157,&quot;title&quot;:&quot;Wtfockdown - Afl 6&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-6&quot;},&quot;pageUuid&quot;:&quot;792d2866-fb20-4f33-b77c-56142384f7d2&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 6&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 6&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;7b0eac94-9c3f-4e57-94a0-5edc79a1e1a4&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902153,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:162,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 5&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/05-qxzqop.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=27e698d9413fb56592012a2c223e0397&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-5&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19744&quot;,&quot;nodeUuid&quot;:&quot;77d2e056-004b-42f2-bbeb-d81fa6a23ccc&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902153,&quot;title&quot;:&quot;Wtfockdown - Afl 5&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-5&quot;},&quot;pageUuid&quot;:&quot;77d2e056-004b-42f2-bbeb-d81fa6a23ccc&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 5&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 5&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;63d45e15-b12d-49b0-be6e-457962fe44e3&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;&quot;,&quot;createdDate&quot;:1610903129,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:124,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 4&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/04-qxzrdg.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=4f83bf8347dcbf8e2d0104b7bc68ff07&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-4&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19750&quot;,&quot;nodeUuid&quot;:&quot;ad6f43b7-dfc4-4b83-99fd-226d3fea446d&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610903129,&quot;title&quot;:&quot;WtfockDown - Afl 4&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-4&quot;},&quot;pageUuid&quot;:&quot;ad6f43b7-dfc4-4b83-99fd-226d3fea446d&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;WtfockDown - Afl 4&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 4&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;4be3ad3d-6750-4455-b8bb-408f87f24aad&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902793,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:200,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 3&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/03-qxzr4a.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=9aaf15a1326c7a2c5fcdbbae794d22a5&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-3&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19747&quot;,&quot;nodeUuid&quot;:&quot;244a8e4b-211b-4b29-86d2-83d7a6e5e749&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902793,&quot;title&quot;:&quot;Wtfockdown - Afl 3&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-3&quot;},&quot;pageUuid&quot;:&quot;244a8e4b-211b-4b29-86d2-83d7a6e5e749&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626557,&quot;title&quot;:&quot;Wtfockdown - Afl 3&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 3&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;4454beeb-cc25-47bc-8fe5-86536921a6bb&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610902711,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:144,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;Wtfockdown - Afl 2&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/02-qxzr1z.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=c62d57fb7e9cecb41b6a822154568e7f&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-2&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19746&quot;,&quot;nodeUuid&quot;:&quot;3637629a-0b09-4be0-8e38-c42794612f20&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610902711,&quot;title&quot;:&quot;Wtfockdown - Afl 2&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-2&quot;},&quot;pageUuid&quot;:&quot;3637629a-0b09-4be0-8e38-c42794612f20&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626556,&quot;title&quot;:&quot;Wtfockdown - Afl 2&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;Wtfockdown - Afl 2&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;5f51a5e3-7997-4627-af45-227466d41551&quot;,&quot;whatsonId&quot;:null},{&quot;autoplay&quot;:false,&quot;badge&quot;:&quot;&quot;,&quot;cimTag&quot;:&quot;vid.wex.frag.vod.free&quot;,&quot;createdDate&quot;:1610892612,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;duration&quot;:282,&quot;embedCta&quot;:null,&quot;enablePreroll&quot;:true,&quot;episodeNumber&quot;:&quot;&quot;,&quot;episodeTitle&quot;:&quot;WtfockDown - Afl 1&quot;,&quot;hasProductPlacement&quot;:false,&quot;image&quot;:&quot;https:\/\/images.goplay.be\/meta\/vlcsnap-2021-08-17-15h28m09s456-qxzot0.png?auto=format&amp;fit=max&amp;h=720&amp;ixlib=php-1.1.0&amp;q=65&amp;w=1280&amp;s=27e2b5012ad80fd4374088a97ee8334b&quot;,&quot;isProtected&quot;:true,&quot;isSeekable&quot;:true,&quot;isStreaming&quot;:false,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown\/wtfockdown-afl-1&quot;,&quot;midrollOffsets&quot;:[],&quot;needs16PlusLabel&quot;:false,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19719&quot;,&quot;nodeUuid&quot;:&quot;ea87a606-e09a-4da3-8c43-c82881e338d9&quot;,&quot;notificationsScore&quot;:17,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1610892612,&quot;title&quot;:&quot;WtfockDown - Afl 1&quot;,&quot;type&quot;:&quot;video-long_form&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown\/wtfockdown-afl-1&quot;},&quot;pageUuid&quot;:&quot;ea87a606-e09a-4da3-8c43-c82881e338d9&quot;,&quot;parentalRating&quot;:&quot;&quot;,&quot;path&quot;:&quot;&quot;,&quot;program&quot;:{&quot;title&quot;:&quot;wtFOCK&quot;,&quot;poster&quot;:&quot;https:\/\/images.goplay.be\/2021-04\/onlinempl1050x1500wtfocks5-qs1ei8.jpg?auto=format&amp;dpr=2&amp;fit=clip&amp;h=290&amp;ixlib=php-1.1.0&amp;q=95&amp;w=200&amp;s=d14666474428d7858651896e6099daf9&quot;,&quot;category&quot;:&quot;Fictie&quot;},&quot;seasonNumber&quot;:&quot;&quot;,&quot;seekableFrom&quot;:1631626556,&quot;title&quot;:&quot;WtfockDown - Afl 1&quot;,&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;,&quot;item_variant&quot;:&quot;WtfockDown - Afl 1&quot;},&quot;type&quot;:&quot;long_form&quot;,&quot;unpublishDate&quot;:&quot;&quot;,&quot;videoUuid&quot;:&quot;00d264fc-c54f-4fc3-b12f-df6629172bbb&quot;,&quot;whatsonId&quot;:null}],&quot;id&quot;:&quot;a6fc4052-b585-4c62-b99c-61e733b5c80d&quot;,&quot;link&quot;:&quot;\/video\/wtfock\/wtfockdown&quot;,&quot;pageInfo&quot;:{&quot;author&quot;:&quot;ken.ceulemans@sbsbelgium.be&quot;,&quot;brand&quot;:&quot;GoPlay&quot;,&quot;description&quot;:&quot;wtFOCK volgt het reilen en zeilen van enkele tienermeisjes en -jongens op de middelbare school en daarbuiten. &quot;,&quot;nodeId&quot;:&quot;19718&quot;,&quot;nodeUuid&quot;:&quot;a6fc4052-b585-4c62-b99c-61e733b5c80d&quot;,&quot;notificationsScore&quot;:0,&quot;program&quot;:&quot;wtFOCK&quot;,&quot;programKey&quot;:&quot;wtfock&quot;,&quot;programId&quot;:&quot;14498&quot;,&quot;programUuid&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;publishDate&quot;:1599145381,&quot;title&quot;:&quot;wtFOCKdown&quot;,&quot;type&quot;:&quot;playlist&quot;,&quot;tags&quot;:[],&quot;unpublishDate&quot;:0,&quot;url&quot;:&quot;https:\/\/www.goplay.be\/video\/wtfock\/wtfockdown&quot;},&quot;title&quot;:&quot;wtFOCKdown&quot;}],&quot;social&quot;:{&quot;facebook&quot;:&quot;https:\/\/www.facebook.com\/wtFOCKbe\/&quot;,&quot;hashtag&quot;:&quot;wtFOCKbe&quot;,&quot;instagram&quot;:&quot;https:\/\/www.instagram.com\/wtfock.be&quot;,&quot;twitter&quot;:&quot;&quot;},&quot;tracking&quot;:{&quot;item_name&quot;:&quot;wtFOCK&quot;,&quot;item_id&quot;:&quot;ee83d837-5cbf-4fd7-8910-a357745a375b&quot;,&quot;item_brand&quot;:&quot;GoPlay&quot;,&quot;item_category&quot;:&quot;Fictie&quot;}}}"></div>
```

Reply:
Oh, okay. The JSON embedded into the HTML is HTML encoded.. 😓

Reply:
Initially, it looks like we can get away with some ugly find/replace.

Otherwise, we could hide `fun String.unescapeHtml()` behind `expect/actual`.